docs/ADD_PKG_MGR.md

```diff
@@ -0,0 +1,22 @@
+# Adding support for a new package manager
+
+Bazel largely relies on existing package managers for each language ecosystem to fetch dependencies.
+Thus the knowledge of provenance and package metadata needs to be plumbed through these tools.
+
+## 
+
+In BUILD targets generated for third-party packages, implementations must create one of the following:
+
+```starlark
+load("@rules_license//rules:license.bzl", "license")
+
+package(
+   default_package_metadata = [":license"],
```

Comment:
If people are doing the integration, we should take the same opportunity to add `package_info` rules to pull package names and versions into the BUILD file in a uniform way.